rs/src/management/http_client.rs

```diff
@@ -456,6 +460,12 @@ impl TunnelManagementClient {
         let headers = request.headers_mut();
         headers.insert("User-Agent", self.user_agent.clone());
 
+        // Add Windows group policies to the header
+        let policy_header_value = self.policy_provider.get_header_value("").await?;
+        if !policy_header_value.is_empty() {
```

Comment:
You should use the Option type here instead of treating empty strings specially. Then you can do something like `if let Some(h) = policy_header_value { /* h is a String here */ }`